src/components/UserInfo/UserInfo.tsx

```diff
@@ -15,41 +15,43 @@ export interface Props {
   field: string;
   career: string;
   mbti: string;
+  title?: string;
 }
 
-function DetailLeaderInfo({
+function UserInfo({
   profileImageUrl,
   nickname,
   field,
   career,
   mbti,
+  title,
 }: Props) {
   return (
-    <LeaderInfoContainer>
-      <Typography variant='h5'>리더 정보</Typography>
-      <LeaderInfoWrapper>
+    <UserInfoContainer>
+      {title && <Typography variant='h5'>{title}</Typography>}
+      <UserInfoWrapper>
         {typeof profileImageUrl === 'string' && profileImageUrl ? (
-          <LeaderProfileImageWrapper>
-            <LeaderProfileImage src={profileImageUrl} alt='profile-image' />
-          </LeaderProfileImageWrapper>
+          <UserProfileImageWrapper>
+            <UserProfileImage src={profileImageUrl} alt='profile-image' />
+          </UserProfileImageWrapper>
         ) : (
           <AccountCircleIcon fontSize='large' color='secondary' />
         )}
```

Comment:
현재 `imageWrapper`가 32, 32px로 되어있습니다! `figma`에서 확인해보니 `Mui Avatar`를 사용하면 크기(40, 40)도 딱 맞추고 따로 래핑하지 않아도 될 것 같아서 아래처럼 수정해도 될 것 같습니다!

```suggestion
          <Avatar src={profileImageUrl} alt='profile-image' /> : <Avatar src='' />
```